Library/Formula/jenkins.rb

```diff
@@ -58,12 +64,12 @@ def caveats; <<-EOS.undent
   test do
     ENV["JENKINS_HOME"] = testpath
     pid = fork do
-      exec "#{bin}/jenkins"
+      exec "java -Djava.io.tmpdir=#{testpath} -jar #{opt_libexec}/jenkins.war"
```

Comment:
@dunn Ok that will work too although if I do it that way, the temp dir only applies to the test. If I pass `"-Djava.io.tmpdir=#{HOMEBREW_TEMP}"` to `bin.write_jar_script()`, it applies to running jenkins too. Wouldn't that be preferable?

In any case, the bigger problem now seems to be that http://mirrors.jenkins-ci.org/war-rc/2.0/jenkins.war seems to be a moving target. The SHA256 has changed twice since this PR was created.